packages/terra-toggle/tests/jest/Toggle.test.jsx

```diff
@@ -21,9 +21,11 @@ describe('Toggle', () => {
   // Prop Tests
   it('should have all props set correctly', () => {
     const toggle = shallow(<Toggle isAnimated isOpen>All props</Toggle>);
```

Comment:
I think we can update this to a screenshot & jest will do this comparison for us